tests/integ/modin/hybrid/test_switch_operations.py

```diff
@@ -661,3 +664,311 @@ def test_set_index():
     # SNOW-2333472: Switching backends then setting the index should be valid.
     s.index = ["a"]
     assert_snowpark_pandas_equal_to_pandas(s, native_pd.Series([0], index=["a"]))
+
+
+def _test_stay_cost(data_obj, api_cls_name, method_name, args, expected_cost):
```

Comment:
```suggestion
def _test_stay_cost(*, data_obj, api_cls_name, method_name, args, expected_cost):
```

There are a lot of parameters here, so I think it's clearer to make them all keyword arguments.

Reply:
that makes sense. I've changed the signature and made them all kwargs